WTWSMS/common/nicknames/VIET_immersion_nicknames.txt

```diff
@@ -221,7 +235,6 @@ nick_the_ever_craving = {
 
 nick_the_anger_filled = {
 	allow = {
-		religion = buddhist
 		OR = {
```

Comment:
Where does the nickname have any tie to Buddhism? Right now, only Buddhists can get it, but being known for anger is an universal recognition considered sinful in most religions. "Anger filled as a Buddhist" is only because the code says that only Buddhists can get it.